openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -244,13 +233,36 @@ def get_placeholder_options(self, options=None):
         return self.get_load_plugin_options(options)
 
     def post_placeholder_process(self, placeholder, failed):
-        """Hide placeholder, add them to placeholder set
-        """
+        """Hide placeholder, add them to placeholder set"""
         node = placeholder.scene_identifier
 
         cmds.sets(node, addElement=PLACEHOLDER_SET)
         cmds.hide(node)
         cmds.setAttr(node + ".hiddenInOutliner", True)
+        model = cmds.ls(assemblies=True)
+
+        filtered_model = []
+        exclude_list = ["persp", "top", "front", "side", "root", "wip"]
+
+        for element in model:
+            if not any(exclude_elem in element for exclude_elem in exclude_list):
+                filtered_model.append(element)
+
+        cameras = cmds.ls(type="camera")
+        renderable_camera = None
+
+        for camera in cameras:
+            if cmds.getAttr(camera + ".renderable"):
+                shapes = cmds.listRelatives(
+                    camera, parent=True, fullPath=True, type="transform"
+                )
+                if shapes:
+                    renderable_camera = shapes[0]
+                    break
+
+        cmds.lookThru(renderable_camera)
+        print("filtered_model: ", filtered_model)
+        cmds.viewFit(filtered_model)
 
```

Comment:
The filter model logic only applies to this - and not to the renderable camera loop above. As such I'd move the logic closer together to make it clear that it's only needed for this and doesn't have anything to do with the renderable camera.